pydream/core.py

```diff
@@ -53,15 +56,24 @@ def run_dream(parameters, likelihood, nchains=5, niterations=50000, start=None,
         parameters = [parameters]
 
     model = Model(likelihood=likelihood, sampled_parameters=parameters)
-    
+
     if restart:
+        model_prefix = kwargs['model_name']
         step_instance = Dream(model=model, variables=parameters,
-                              history_file=kwargs['model_name'] + '_DREAM_chain_history.npy',
-                              crossover_file=kwargs['model_name'] + '_DREAM_chain_adapted_crossoverprob.npy',
-                              gamma_file=kwargs['model_name'] + '_DREAM_chain_adapted_gammalevelprob.npy',
+                              history_file=model_prefix + '_DREAM_chain_history.npy',
```

Comment:
Does it matter that the current working directory is used and it's not configurable? I know this was existing behaviour, just wondering your thoughts.

Reply:
My approach has been to make as few changes as possible (probably not the best approach :/ ). Since there haven't been any issues or complaints before I wouldn't think it's that important to make it configurable. If you think it would be a useful feature I would be happy to do it

Reply:
You're right, it's generally a good idea to keep PRs small and focussed. Could be addressed separately.